Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Payment Flow/PaymentFormViewController.m

```diff
@@ -81,7 +77,16 @@ - (void)completePayment {
             UIAlertController *alert = [Utils createAlertControllerWithTitle:@"Could not save payment to Parse server." andMessage:error.localizedDescription okCompletion:nil cancelCompletion:nil];
             [self presentViewController:alert animated:YES completion:nil];        }
     }];
+}
+
+- (void)updateBasketFeaturedValueWeights {
+    FeaturedValueWeight *ftWeights = self.basket.featuredValueWeights;
+    NSUInteger maxNumberOfDonationsWeight = 30;
+
+    ftWeights.numberOfDonationsWeight = MIN(self.basket.allTransactions.count, maxNumberOfDonationsWeight);
+    NSInteger sumFeaturedVal = ftWeights.numberOfDonationsWeight + ftWeights.predeterminedEventRelevancyWeight + ftWeights.userFavoritesWeight;
 
+    self.basket.totalFeaturedValue = [[NSNumber alloc] initWithLong:sumFeaturedVal];;
```

Comment:
This means all baskets with more than 40 donations and likes are all ranked equally and only differentiated by the `predeterminedEventRelevancyWeight`, is that what you want?